src/globalvars.cpp

```diff
@@ -13,9 +13,46 @@
 
 namespace bpftrace::globalvars {
 
-void update_global_vars(const struct bpf_object *bpf_object,
-                        struct bpf_map *global_vars_map,
-                        BPFtrace &bpftrace)
+const GlobalVarConfig &get_config(GlobalVar global_var)
+{
+  auto it = GLOBAL_VAR_CONFIGS.find(global_var);
+  if (it == GLOBAL_VAR_CONFIGS.end()) {
+    LOG(BUG) << "Unknown global variable " << to_string(global_var);
```

Comment:
`to_string` calls `get_config` right so this wouldn't work

Reply:
yup, will fix, thanks!